ITRStartTask/AppLibrary/ApiService/ImageDownloader/ImageDownloadOperation.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ImageDownloadOperation.swift
+//  ITRStartTask
+//
+//  Created by Homyakov, Ilya2 on 30/08/2018.
+//  Copyright © 2018 Homyakov, Ilya2. All rights reserved.
+//
+
+import UIKit
+
+class ImageDownloadOperation: AsyncOperation {
+    enum ImageDownloadOperationError: Int {
```

Comment:
Need to think how to avoid code doubling

Reply:
Fixed